cub/test/test_device_radix_sort.cu

```diff
@@ -1875,10 +1875,10 @@ void TestUnspecifiedRanges()
 
       for (std::size_t sid = 0; sid < max_segments; sid++)
       {
-        const int segment_size = 
+        const int segment_size =
           static_cast<int>(RandomValue(avg_segment_size));
 
-        const bool segment_is_utilized = segment_size > 0 
+        const bool segment_is_utilized = segment_size > 0
```

Comment:
suggestion: depending on when you plan to add the follow up PR, I'd recommend disabling non-segmented test and small problem sizes here.

Reply:
I plan to finish the rest of the tests this week. If that doesn't happen, I'll look into doing some surgery here to reduce the old coverage as an intermediate step.